manifests/charts/istiod-remote/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+name: istiod-remote
+version: 1.1.0
```

Comment:
please change the version to 1.6.0. We have, or had at one point, as part of the release process an automated sed of version and appVersion fields from 1.6.0 to the latest release.

Reply:
Can we leave this version thing for another issue discussion?  the main istiod chart is using 1.1.0 so I don't want to have have remote move to 1.6.0. :)

Reply:
@howardjohn I don't recall how you handled the chart versioning in the release of the manifests. In the past, we had a sed operation that did this work. I even wrote the sed - a year or more ago. That was replaced by the genbuilder tool. I am not clear how or if genbuilder modifies the output to version the charts.